helm/cert-manager-app/templates/deployment.yaml

```diff
@@ -37,7 +37,5 @@ spec:
                 fieldPath: metadata.namespace
           resources:
             requests:
```

Comment:
For limits to be configurable too via `values.yaml`, whole `.Values.resources` has to be included here with `toYaml` + indent (or nindent) as done in nginx https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/helm/nginx-ingress-controller-app/templates/controller-deployment.yaml#L126

Learned recently about even better (cleaner template) alternative, use toJson https://twitter.com/tsaha/status/1252317355457122304